src/js/components/modals/AppModalComponent.jsx

```diff
@@ -68,6 +69,7 @@ var AppModalComponent = React.createClass({
     AppFormStore.on(FormEvents.CHANGE, this.onFormChange);
     AppFormStore.on(FormEvents.FIELD_VALIDATION_ERROR,
       this.onFieldValidationError);
+    Mousetrap.bind(["command+enter", "ctrl+enter"], () => this.handleSubmit());
```

Comment:
How about this:



<img width="747" alt="screen shot 2016-02-05 at 11 38 49" src="https://cloud.githubusercontent.com/assets/1078545/12843998/0edc67b6-cbfd-11e5-92f9-734bff9ef363.png">
?
cc @leemunroe